src/09-workorders/work-orders.apib

```diff
@@ -1,5 +1,74 @@
 ## Work Orders [/workOrders]
 
+### workOrders.list [GET /workOrders.list]
+
+Get a list of work orders.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + customer (object, optional)
+                + type (enum[string], required)
+                    + Members
+                        + contact
+                        + company
+                + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+            + state (enum[string], optional)
+                + Members
+                    + draft
+                    + finalized
+                    + sent
+        + page (Page, optional)
+        + sort (array, optional)
```

Comment:
Not sure if sorting is in scope as well. We could still create a separate story or already include it in this one.

Reply:
Pagination and sorting were added to the same user story.